docs/strat-lib/getting-started/smart-offer.md

```diff
@@ -270,28 +204,36 @@ cast send --rpc-url $LOCAL_URL "$WBTC" "mint(uint)" 500000000 --private-key "$PR
 If the admin acts as taker and takes the offer (see [snipe guide](../guides/howToSnipe.md)), you will also need USDT.
 
 ```bash
-cast send --rpc-url $LOCAL_URL "$USDT" "mint(uint)" 100000000000 --private-key "$PRIVATE_KEY"
+cast send --rpc-url $LOCAL_URL "$USDT" "mint(uint)" 1000000000000 --private-key "$PRIVATE_KEY"
```

Comment:
I intentionally reduced the number of 0s because otherwise minting failed for me. If it works now then OK - I think the token may have some mint limits

Reply:
Yep the idea was to mint 100k USDT, some zeros must have stayed when whanging token symbols - I'll remove 2 of them